app/index.js

```diff
@@ -120,6 +161,40 @@ function emitRewardedOnCycle() {
   })
 }
 
+async function emitRegistry() {
+  try {
+    logger.info('emitRegistry')
+    const currentBlock = (await web3.eth.getBlockNumber()).toNumber()
+    const chains = await blockRegistry.methods.getRpcs.call()
```

Comment:
should be getRpcs()
and that's why I want unit tests or testnet deployment with mock contracts to see that this whole process is working